README.md

```diff
@@ -27,7 +27,7 @@ projects. This tool does not only detect them, but also fixes them for you.
 * PHP 8.3
 * PHP 8.4
 
-> **Note**
+> [!NOTE]
```

Comment:
Left is old style, right is new style:

<img width="1780" height="392" alt="Image" src="https://github.com/user-attachments/assets/91d93e5a-fc26-4886-aeed-bc58d7e61fc0" />